static/app/routes.tsx

```diff
@@ -1193,6 +1193,15 @@ function routes() {
               }
               component={errorHandler(LazyLoad)}
             />
+            <Route
+              path="manage/"
+              componentPromise={() =>
+                import(
+                  /* webpackChunkName: "WidgetNew" */ 'app/views/dashboardsV2/manage'
```

Comment:
This webpack chunk name should be unique.

```suggestion
                  /* webpackChunkName: "ManageDashboards" */ 'app/views/dashboardsV2/manage'
```

Reply:
A button to access this new page would be nice to have  👍